speechbrain/lobes/models/huggingface_whisper.py

```diff
@@ -131,18 +138,29 @@ def forward(self, wav, decoder_input_ids=None):
                 out_encoder = self.forward_encoder(wav)
                 if self.encoder_only:
                     return out_encoder
-                logits, attn = self.forward_decoder(
-                    out_encoder, decoder_input_ids
-                )
+
+                if self.output_all_hiddens:
+                    logits, attn = self.forward_decoder(
+                        out_encoder[-1, ...], decoder_input_ids
```

Comment:
From my understanding, `out_encoder[-1, ...]` is exactly the same as `out_encoder[-1]`, so why adding "..." ?